pkg/certificates/agent/reconciliation/cert_agent_reconciler.go

```diff
@@ -120,18 +125,21 @@ func (r *certAgentReconciler) reconcile(_ ezkube.ResourceId) (bool, error) {
 	return false, errs
 }
 
-func (r *certAgentReconciler) reconcileIssuedCertificate(
-	issuedCertificate *v1.IssuedCertificate,
-	inputSecrets corev1sets.SecretSet,
-	inputPods corev1sets.PodSet,
-	inputConfigMaps corev1sets.ConfigMapSet,
-	inputCertificateRequests v1alpha2sets.CertificateRequestSet,
-	podBounceDirectives v1alpha2sets.PodBounceDirectiveSet,
+// Exposed for testing
+func (r *certAgentReconciler) ReconcileIssuedCertificate(
+	issuedCertificate *certificatesv1.IssuedCertificate,
+	inputSnap input.Snapshot,
 	outputs certagent.Builder,
 ) error {
+
+	// If this translator doesn't own the resource, return early
```

Comment:
Let's make this comment more specific as to why the cert agent isn't responsible for processing this IssuedCertificate. Something to the effect of,

```suggestion
	// If the IssuedCertificate's IssuedCertificateSecret is nil, the cert-agent is not responsible for issuing the certificate.
```